fastlane/lib/fastlane/version.rb

```diff
@@ -1,3 +1,3 @@
 module Fastlane
-  VERSION = '1.80.0'.freeze
+  VERSION = '1.80.1'.freeze
```

Comment:
Please don't include a version bump in the PR, as this would lead to conflicts with other PRs if everyone does it